packages/create-react-app/createReactApp.js

```diff
@@ -0,0 +1,464 @@
+'use strict';
```

Comment:
Can you rewrite the history so that you first move the file (`index.js` -> `createReactApp.js`) and commit that alone, and then make more changes in further commits? This way we can preserve the blame history for the file.